webapp/src/main/webapp/vue-apps/sidebar/components/drawer/SidebarThirdLevel.vue

```diff
@@ -23,13 +23,13 @@
     ref="thirdLevelDrawer"
     v-model="drawer"
     :width="drawerWidth"
-    :style="drawerOffsetStyle"
+    :style="`background: inherit; ${drawerOffsetStyle}`"
     :right="$vuetify.rtl"
     class="HamburgerMenuThirdLevelParent border-box-sizing z-index-drawer"
     max-width="100%"
     hide-overlay>
     <v-hover v-if="drawer" v-model="$root.hoverThirdLevel">
-      <div class="full-width fill-height overflow-x-hidden overflow-x-auto specific-scrollbar">
+      <div class="full-width fill-height overflow-x-hidden overflow-x-auto specific-scrollbar transparent">
```

Comment:
Not needed, isn't it ?

```suggestion
      <div class="full-width fill-height overflow-x-hidden overflow-x-auto specific-scrollbar">
```

Reply:
done!